src/updateDisplay.js

```diff
@@ -0,0 +1,26 @@
+import pressedAnotherOperator from "./pressedAnotherOperator";
+
+const updateDisplay = (result, key, hasLastChar, lastEquation) => {
+  var tempResult = result;
```

Comment:
I replaced currentBtn to key and tempResult to result within the `updateDisplay` function and everything works fine. I thought they needed to be separated but it doesn't look like it. Although I cannot use 'result' in App.js to replace tempResult in the destructuring because result has already been declared. Perhaps I can use currentResult? What do you think?

Reply:
I'm only concerned about the function internals. App.JS can stay the same

Reply:
Ah okay, got it.